apidoc2swagger/apidoc2swagger.php

```diff
@@ -109,7 +133,7 @@ private function createResponseReference($_apiCall)
                 "description" => "Standard response for successful HTTP requests. The actual response will depend on the request method used. In a GET request, the response will contain an entity corresponding to the requested resource. In a POST request, the response will contain an entity describing or containing the result of the action.",
                 "schema" => array(
                     "title" => $name."s",
-                    "type" => "array"
+                    "type" => "string"
```

Comment:
Yes because the type refers to each response in the response array which are Strings.

This is how it looks in yaml:
![image](https://cloud.githubusercontent.com/assets/13750648/25226824/92527dfa-25c6-11e7-9092-0e53f9e66a26.png)

Example response:
![image](https://cloud.githubusercontent.com/assets/13750648/25226882/c3c55e52-25c6-11e7-9296-55892006ac43.png)
As you can see, the <message> tags contain a string.
Technically the <success> tags contain a number, but since numbers can be treated as strings, i didn't bother to write code that parses the request examples, checks for a number and then changes the type. The effort exceeds the reward here imo.